spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+# rubocop:disable Metrics/BlockLength
+describe User, type: :feature do
+  before :each do
+    @user1 = User.create(name: 'Ema',
+                         photo: 'https://images.unsplash.com/photo-1661956602116-aa6865609028?ixlib=rb-4.0.3&ixid=MnwxMjA3fDF8MHxlZGl0b3JpYWwtZmVlZHwxfHx8ZW58MHx8fHw%3D&auto=format&fit=crop&w=500&q=60',
+                         bio: 'Here my story',
+                         posts_counter: 3)
+  end
+
+  it 'should have username Ema' do
+    visit user_path(@user1)
+    expect(page).to have_content('Ema')
+  end
+
+  it 'should display number of posts by Ema' do
+    visit user_path(@user1)
+    Post.create(author: @user1, title: 'Introduction to rails', text: 'Rails is a web', comments_counter: 0,
+                likes_counter: 0)
+    Post.create(author: @user1, title: 'Introduction to testing', text: 'Rails is a web', comments_counter: 0,
+                likes_counter: 0)
+    Post.create(author: @user1, title: 'Introduction to Capybara', text: 'Rails is a web', comments_counter: 0,
+                likes_counter: 0)
+    expect(page).to have_content('3 Posts')
+  end
```

Comment:
- [ ] This test is failing. Kindly look into it and fix it.

  ![djue](https://user-images.githubusercontent.com/36542986/210368479-667f2fc8-2d66-4cff-aaa0-15fc41992531.jpg)